exit-what-1.md

```diff
@@ -3,6 +3,7 @@ layout: default
 title: Exit What 1
 ---
 
-# Exit what 1
+# Things to know:
 
-Explanation of why not to proceed after the first what question.
+- If you are requesting access to a copy of your health insurance information, your health plan or provider has up to 30 days (sometimes more) to provide the information. Your health insurance plan or provider may require that you make your request in writing.
+- Learn more about your right to access your medical record [here](https://www.hhs.gov/hipaa/for-individuals/medical-records/index.html).
```

Comment:
Let's make this whole sentence a link (without the period), instead of just "here." In general, avoid linking just the word "here."